R/load.r

```diff
@@ -158,6 +158,12 @@ load_all <- function(path = ".", reset = TRUE, compile = NA,
       # methods. We'll restore the foreign methods but let the package
       # register its own methods again.
       old_methods <- as.list(methods_env)
+
+      is_foreign_method <- function(x) {
```

Comment:
Can you define this in the top level rather the in the body of the function?

Reply:
done; I thought this definition wasn't general enough to be at top level.

Reply:
That is fair, I just as a rule avoid defining functions within other function bodies, as it makes it hard to test them in isolation.